.github/workflows/dapr.yml

```diff
@@ -30,17 +30,150 @@ on:
       - release-*
       - feature/*
 jobs:
-  build:
-    name: "Build and test on ${{ matrix.job_name }}"
-    runs-on: "${{ matrix.os }}"
+  lint:
+    name: lint & proto validation
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target_os: ["linux"]
```

Comment:
I keep this because it is easier to reuse steps that reference matrix.target_os and avoids bugs of copy and pasting. I can remove if anyone feels strong about this.